index.js

```diff
@@ -198,15 +198,17 @@ module.exports = {
 	/**
 	 * Generates an intent request object.
 	 * @param {string} intentName The name of the intent to call.
-	 * @param {object} slots Slot data to call the intent with.
+	 * @param {object} requestSlots Slot data to call the intent with.
```

Comment:
Why did you rename the param?

Reply:
Well, I think I needed to rename the param name or the variable inside the method in order to make this work. I decided to rename the param to avoid changing the code too much.

Reply:
Generally, I think renaming code inside a function is better as nobody relies on it in any way.

Reply:
Well, I think either way works for this specific scenario. I'll change the name of the variable inside the function instead of the param name if that's better for you, no problem 👍